modules/news/news.model.js

```diff
@@ -1,12 +1,11 @@
 const mongoose = require('mongoose');
 const Language = require('../../models/Language').schema;
-const PrimaryImage = require('../../models/PrimaryImage').schema;
 const ImageSet = require('../common/ImageSet').schema;
 
 const NewsSchema = new mongoose.Schema({
   title: [Language],
   text: [Language],
-  images: PrimaryImage,
+  images: ImageSet,
```

Comment:
We have only one image, so you can add image uri string and set `image: String`.